tests/recover_deadlock.test/lrl.options

```diff
@@ -0,0 +1 @@
+sql_tranlevel_default blocksock
```

Comment:
This test tests that a bunch of selects that run during schema changes succeed because they go through the recover ddlk process correctly. This is not relevant for snapshot since these selects will fail.